data/authors/charles.mdx

```diff
@@ -0,0 +1,26 @@
+---
+name: Charles C.
+nym: charles
+avatar: /static/images/ops/charles.jpg
+occupation: Chief Fiat Officer
+company: OpenSats
+ops: true
+---
+
+Charles might strike the uninitiated as a very boring man. As the son of an
+accountant and a librarian turned bureaucrat, he grew up immersed in a world of
+spreadsheets and paper-pushing. But he is not a boring man. He has a very
+particular set of skills; skills he has acquired over a very long career. Skills
+that make him effective at navigating the nightmare of bureaucracy.
+
+As Chief Fiat Officer, Charles takes care of—surprise, surprise—the "fiat" side
```

Comment:
@dergigi and @Arvin21M I spent the last 10 min trying to find the thread where we discussed about how to make suggestions directly into the files so that they can be easily "accepted" or "declined". It seems to be a great resource for most people joining now. I used the search box on the top bar but could not find it. Any suggestion on how to improve my search skills on GitHub is much appreciated.

@bitcoinprecept once we find this, I'll link it here for everyone in the @OpenSats/volunteers group as it's very useful for making our revisions. I bang my head against the wall for a few days to learn this, and then @dergigi and @Arvin21M rescued (taught) me. But now I need to learn how to search stuff in GitHub before I can help others :(

Reply:
@lucasmogu & @OpenSats/volunteers I think you are referring to this Haacked article on easily making change requests:

https://haacked.com/archive/2019/06/03/suggested-changes/

Reply:
Thanks @Arvin21M that's exactly it!